cloud/organization/organization.go

```diff
@@ -0,0 +1,174 @@
+package organization
+
+import (
+	http_context "context"
```

Comment:
```suggestion
     "context"
```

Not sure why we need to alias the import, but in case there is a need to better name it as `httpContext` so as to follow the golang naming convention